analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.TryCatchFinally.cs

```diff
@@ -435,5 +436,178 @@ public void Finally_NestedFinally()
                 "AfterInnerFinally",
                 "AfterOuterFinally");
         }
+
+        [TestMethod]
+        public void TryCatch_ThrowInTry_SingleCatchBlock()
+        {
+            const string code = @"
+Tag(""BeforeTry"");
+try
+{
+    Tag(""InTry"");
+    throw new System.Exception();
+    Tag(""UnreachableInFinally"");
+}
+catch
+{
+    Tag(""InCatch"");
+}
+Tag(""AfterCatch"");";
+            SETestContext.CreateCS(code).Validator.ValidateTagOrder(
+                "BeforeTry",
+                "InTry",
+                "InCatch",          // With Exception thrown by Tag("InTry")
+                "InCatch",          // With Exception thrown by `throw`
```

Comment:
Maybe assert the comments here too, like in the comment from Finally_ThrowInTry (and not anywhere else, yet)